cmd/cosign/cli/verify/verify_blob_attestation.go

```diff
@@ -140,32 +140,17 @@ func (c *VerifyBlobAttestationCommand) Exec(ctx context.Context, artifactPath st
 	}
 
 	// Set up TSA, Fulcio roots and tlog public keys and clients.
-	if c.RFC3161TimestampPath != "" && c.KeyOpts.TSACertChainPath == "" {
+	if c.RFC3161TimestampPath != "" && c.TSACertChainPath == "" {
```

Comment:
I noticed we inconsistently use `c.KeyOpts.TSACertChainPath` vs `c.TSACertChainPath`. Can we standardize that across the verify_* impls?